src/ModbusBackend.cc

```diff
@@ -76,17 +76,20 @@ namespace ChimeraTK {
         throw ChimeraTK::runtime_error(std::string("ModbusBackend: Connection failed: ") + modbus_strerror(error));
```

Comment:
Not sure how the new Exception handling works. It looks like you throw an exception even after setOpenedAndClearException(). I think the purpose is to show the message in the Device error. In this case the first exception (this here) will not be visible as Device error right?
So should setOpenedAndClearException() be called before?

Reply:
Not sure what you mean exactly. There are two cases:

1. The `open()` is called for the very first time on this backend instance. In that case the backend would stay in the closed state instead of going into the exception state, which is barley distinguishable, but I have added a `setException()` call now to fix that.
2. `open()` is called after there was an exception to clear the exception state. In this case we do not want to overwrite the original exception message with a new message that merely tells you that you still cannot open the device, because it will hide the original message in case of an automatic recovery procedure (especially in ApplicationCore). This would make debugging harder.